### PR TITLE
CTX-4421: Added cutadaptTrim command

### PR DESCRIPTION
**Pull request by VukManojlovic**

### CODE REVIEW

File: coretex/bioinformatics/__init__.py [hunk 1]
```diff
@@ -15,4 +15,35 @@
 #     You should have received a copy of the GNU Affero General Public License
 #     along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
-from .utils import CommandException
+from typing import Optional
+
+from .utils import CommandException, command
+
+
+def cutadaptTrim(
```
Comment (dule1322) [inline coretex/bioinformatics/__init__.py:23]: function is missing documentation

File: coretex/bioinformatics/__init__.py [hunk 2]
```diff
@@ -15,4 +15,35 @@
...
+    forewardOutput: str,
+    forewardAdapter: str,
+    reverseFile: Optional[str] = None,
+    reverseOutput: Optional[str] = None,
+    reverseAdapter: Optional[str] = None
+) -> None:
+
+    args: list[str] = [
+        "cutadapt",
+        "-o", forewardOutput,
```
Comment (dule1322) [inline coretex/bioinformatics/__init__.py:34]: forward is misspelled

File: coretex/bioinformatics/__init__.py [hunk 3]
```diff
@@ -15,4 +15,35 @@
...
 #     along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
-from .utils import CommandException
+from typing import Optional
+
+from .utils import CommandException, command
+
+
+def cutadaptTrim(
+    forewardFile: str,
```
Comment (dule1322) [inline coretex/bioinformatics/__init__.py:24]: why are we using `str` instead of `Path` for paths?

Comment (VukManojlovic) [reply coretex/bioinformatics/__init__.py:24]: Requiring strings is the convention that all other command wrapper functions in the bioinformatics subdirectory use, so this one should not be an exception

Comment (dule1322) [reply coretex/bioinformatics/__init__.py]: For now I will resolve this but this needs to be changed

**Pull request merged**